app_book/views.py

```diff
@@ -32,10 +35,34 @@ def check_and_call(request, *args, **kwargs):
             return HttpResponseRedirect(request.META.get('HTTP_REFERER', '/'))
 
         return func(request, *args, **kwargs)
+
     return check_and_call
 
 
-StatusAndTwoForms = collections.namedtuple("StatusAndTwoForms", ["status", "form_one", "form_two"], verbose=False, rename=False)
+@transaction.atomic
+def change_ownership(book_id, to_user_id):
```

Comment:
Sollte in app_book/services.py ausgelagert werden, weil diese Method nicht von urls.py aufgerufen wird.